eng/versioning/version_client.txt

```diff
@@ -35,7 +35,7 @@ com.azure:azure-communication-common;1.0.0-beta.2;1.0.0-beta.3
 com.azure:azure-communication-administration;1.0.0-beta.2;1.0.0-beta.3
 com.azure:azure-communication-sms;1.0.0-beta.2;1.0.0-beta.3
 com.azure:azure-core;1.10.0;1.11.0-beta.1
-com.azure:azure-core-amqp;1.6.0;1.7.0-beta.1
+com.azure:azure-core-amqp;1.6.0;1.7.0-beta.2
```

Comment:
I'm curious why we are bumping this number here?

Reply:
yes https://search.maven.org/artifact/com.azure/azure-core-amqp/1.7.0-beta.1/jar